manim/mobject/geometry.py

```diff
@@ -597,12 +599,12 @@ class Arrow(Line):
         "preserve_tip_size_when_scaling": True,
     }
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, tip_shape=None, **kwargs):
```

Comment:
79757610679448cc0560ccffc9578d0254b5133c

Reply:
I don't see a nice way to specify `ArrowTriangleTip` directly as the default value for `TipableVMobject.add_tip` (without some serious reordering I always get a `NameError` as `ArrowTriangleTip` is only defined later in the file -- and moving the implementation of the arrow tips before `TipableVMobject` is not possible because of `ArrowCircleTip`, which is a `Circle`, which is an `Arc`, which is a `TipableVMobject` again.)

Maybe this would work if the arrow tips were moved to a separate file -- but I think this might be overkill. I suggest simply handling the case of `tip_shape is None` (as it was before 7975761). Fixed via 249c2841.

Reply:
Perfect, agreed on all accounts